src/components/DayPickerRangeController.jsx

```diff
@@ -668,8 +675,6 @@ export default class DayPickerRangeController extends React.PureComponent {
     } else {
       onDatesChange({ startDate, endDate });
     }
-
-    onBlur();
```

Comment:
removing this call is a breaking change

Reply:
But blur event from `CalendarDay.jsx` is being captured but this(`<div onBlur={this.onBlur}>`) in `DayPickerRangeController.jsx`

Reply:
the reason for `onBlur` call inside `onDayClick` function in `DayPickerRangeController` is because of [this](https://github.com/airbnb/react-dates/blob/master/src/components/CalendarDay.jsx#L160).
We can pass custom `renderCalendarDay` which might not trigger blur event and on day click may unnecessary call `onBlur` if placed inside of `onDayClick`